target/linux/meson/Makefile

```diff
@@ -0,0 +1,35 @@
+# SPDX-License-Identifier: GPL-2.0-or-later
+#
+# Copyright (C) 2017-2023 shiyu1314 <1052672227@qq.com>
+
+include $(TOPDIR)/rules.mk
+
+ARCH:=arm
+BOARD:=meson
+BOARDNAME:=Amlogic Meson family
+FEATURES:=audio boot-part ext4 fpu legacy-sdcard squashfs usbgadget
+MAINTAINER:=shiyu1314 <1052672227@qq.com>
+SUBTARGETS:=meson8b
+
+KERNEL_PATCHVER:=5.15
+KERNEL_TESTING_PATCHVER:=6.1
```

Comment:
I would directly jump for 6.1. Is it problematic? Do we need 5.15 for any reason?

Reply:
No problem, one more choice

Reply:
No problem, one more choice

Reply:
> I would directly jump for 6.1. Is it problematic? Do we need 5.15 for any reason?

No problem, one more choice